src/main/scala/mesosphere/marathon/core/task/termination/KillReason.scala

```diff
@@ -34,6 +34,9 @@ object KillReason {
   /** The task is killed because it is unknown */
   case object Unknown extends KillReason
 
+  /** The task is killed because marathon has a different launchCount for the task of the respective instance */
+  case object Outdated extends KillReason
```

Comment:
Glad to see my suggestion be implemented so fast, but it was really a question. Thinking about it now, `Diverged` might be better – in the case we encountered, the update was actually for a _newer_ attempt, so it was the taskId in the persistence layer that was outdated, not the task that should have been killed.

@meln1k, @timcharper, does `Diverged` make sense to you?

Reply:
@meichstedt how about `NotInSync`?